packages/gatsby/src/utils/worker/render-html.ts

```diff
@@ -156,12 +157,12 @@ async function getScriptsAndStylesForTemplate(
     //     }
     //   }
     // }
-    for (const [rel, assets] of Object.entries(namedChunkGroup.childAssets)) {
-      // @ts-ignore TS doesn't like that assets is not typed and especially that it doesn't know that it's Iterable
-      for (const asset of assets) {
-        handleAsset(asset, rel)
-      }
-    }
+    // for (const [rel, assets] of Object.entries(namedChunkGroup.childAssets)) {
+    //   // @ts-ignore TS doesn't like that assets is not typed and especially that it doesn't know that it's Iterable
+    //   for (const asset of assets) {
+    //     handleAsset(asset, rel)
+    //   }
+    // }
```

Comment:
new `webpack.stats.json` don't have `childAssets`, so:
```
import(
  /* webpackChunkName: "imp-prefetch", webpackPrefetch: true */ `../components/imp-prefetch`
).then(mod => {
  mod.wat()
})
import(
  /* webpackChunkName: "imp-preload", webpackPreload: true */ `../components/imp-preload`
).then(mod => {
  mod.wat2()
})
```

currently is not genering preload/prefetch scripts in .html files